docs/user_documentation.md

```diff
@@ -78,20 +78,22 @@ Then start tensorboard and point it to the model directory (or any parent direct
 ### CPU/GPU training
 
 By default, training is carried out on the first GPU device of your machine.
-You can specify alternative devices with the `--device-ids` option, with
+You can specify alternative GPU devices with the `--device-ids` option, with
 which you can also activate multi-GPU training (see below). If
 `--device-ids -1`, sockeye will try to find a free GPU on your machine and block
-until one is available. The locking mechanism is based on files and therefore assumes all processes are running
+until all of them are available. The locking mechanism is based on files and therefore assumes all processes are running
```

Comment:
Thanks, I forgot to go back to the original version in this part of the sentence. I initially had something similar to what you suggest, but the documentation first covers single-device training and then turns to multi-device. Will fix the sentence.